additional-data/scripts/update-timeline-stats.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * Process baseline data from the web-features package to create
+ * data tables for visualization purpose.
+ *
+ * Some notes:
+ * - Timeline stats do not include features that have not shipped anywhere
+ * because these aren't associated with any date in web-features (32 features
+ * as of January 2025).
+ * - Process drops occurrences of '≤' in dates, because it's hard to deal with
+ * uncertainties in stats.
+ */
+
+import { features, browsers } from "web-features";
+import fs from "node:fs/promises";
+import path from "node:path";
+
+const OUTPUT_TIMELINE = path.join(import.meta.dirname, "..", "timeline-number.json");
+const OUTPUT_DURATIONS = path.join(import.meta.dirname, "..", "timeline-durations.json");
+
+async function main() {
+  // Convert features to an array and pretend all dates are exact 
+  const simplifiedFeatures = Object.entries(features)
+    .map(([id, feature]) => Object.assign({ id }, feature))
+    .map(feature => {
+      if (feature.status.baseline_low_date &&
+          feature.status.baseline_low_date.startsWith("≤")) {
```

Comment:
This is making me realize that we're asking consumers to have to deal with this character over and over again. I wonder if we should think about adding an extra field to the web-features package that omits this character, like:

```json
{
  "baseline_low_date": "≤2022-01-01",
  "baseline_low_date_data": {
    "when": "on | before",
    "data": "2022-01-01"
  }
}
```

Something like the above, where you can either use the same field we've always had, or a more easily machine readable field that has just the absolute date, and another field that indicates if the feature was supported on or before that date.